app/views/transaction/_country_notice.html.erb

```diff
@@ -1,4 +0,0 @@
-<div class="country-notice">
-  <dt class="country-notice__pretext">Applies to:</dt>
```

Comment:
Whoops I missed that this was invalid HTML 🤦

Reply:
> Nice one

Thanks for the review Kevin!